### DISCUSSION TITLE
2806 - Conecção ao banco de dados

### DISCUSSION THREAD

**Discussion by broeringvictor** [balta-io/forum#691] [Tira Dúvidas] [answered]
Opa, professor! Boa tarde! 

Ao executar ao conectar ao banco da forma que o Sr. ensinou não conectava de jeito nenhum.

`"const string connectionString = "Server=localhost,1433;Database=balta;User ID=sa;Password=1q2w3e4r@#$";"`

**Ao recorrer ao ChatGPT ele me falou para colocar um ;TrustServerCertificate=True; e deu certo. Contudo, o que eu posso ter feito de errado na instalação para dar esse tipo de erro? Tem algum problema no SSL do meu computador?** 

`const string connectionString = "Server=localhost,1433;Database=balta;User ID=sa;Password=1q2w3e4r@#$;TrustServerCertificate=True;";`

Eu consegui chegar no no erro ao seguir o tutorial do chet gpt de depuração:
`Erro ao conectar ao banco de dados: A connection was successfully established with the server, but then an error occurred during the login process. (provider: SSL Provider, error: 0 - The certificate chain was issued by an authority that is not trusted.)
`

**Comment by andrebaltieri** [comment, marked as answer]
Bom dia, @broeringvictor, como vai?
Não é um problema é um recurso de segurança... Isto mudou nas versões mais recentes do SQL Server... é uma obrigatoriedade dele.

**Comment by jeffersonguilherme** [comment]
Cara, tava com o mesmo problema, achei que mais alguém teria esse problema, por sorte achei o seu post rapido e com a solução. Obrigado :D
